spec/ddtrace/contrib/httparty/example_client_spec.rb

```diff
@@ -0,0 +1,83 @@
+require 'spec_helper'
+require 'ddtrace'
+require 'httparty'
+
+RSpec.describe 'ExampleClient' do
+  let(:tracer) { get_test_tracer }
+  let(:configuration_options) { { tracer: tracer } }
+
+  before do
+    Datadog.configure do |c|
+      c.use :httparty, configuration_options
+    end
+
+    WebMock.disable_net_connect!
+    WebMock.enable!
+  end
+
+  let(:klass) do
+    class FooClient
+      include HTTParty
+      base_uri 'https://example.com'
+      dd_options service_name: 'foo_service'
+
+      def foo
+        self.class.get('/foo')
+      end
+    end
+    FooClient
```

Comment:
We try to ensure we don't add new constants to the global name.
In this regard, I suggest creating a new class object inside this `let` block and returning it.

```ruby
let(:klass) do
  Class.new do
    include HTTParty
    base_uri 'https://example.com'
    # ...
  end
end
```

Reply:
I caught this one too :). There shouldn't be any warnings now.